README.md

```diff
@@ -317,7 +307,9 @@ az ad sp delete --id $AZ_APP_ID
 
 Create a `Repository` called `platform-ref-azure` in your Upbound Cloud `Organization`:
 
-![Upbound Repository](docs/media/repository.png)
+![](docs/media/2022-01-12-15-56-26.png)
```

Comment:
can we give these media files more descriptive file names, instead of timestamps? e.g. `01-repository-empty.png` or similar

Reply:
also, are all the new media files used? i only see 2 of them referenced during a quick scan

Reply:
> also, are all the new media files used? i only see 2 of them referenced during a quick scan

They are not, they were used in a different, non-committed iteration. I will delete unused.

Reply:
> can we give these media files more descriptive file names, instead of timestamps? e.g. `01-repository-empty.png` or similar

I will, I need to reconfigure my vsc extension that auto-names those.

Reply:
I believe it's good now.